main.py

```diff
@@ -28,8 +29,10 @@
                  methods=["GET"])
 app.add_url_rule(rule="/profile/session/delete", endpoint="profile.main.session_delete",
                  view_func=profile_main.session_delete, methods=["POST"])
-app.add_url_rule(rule="/workspaces", endpoint="profile.main.workspaces", view_func=profile_main.workspaces,
-                 methods=["GET", "POST"])
+app.add_url_rule(rule="/workspaces", endpoint="profile.main.workspaces", view_func=profile_main.workspaces_list_handler,
+                 methods=["GET"])
+app.add_url_rule(rule="/workspaces", endpoint="profile.workspaces.workspaces_list_handler", view_func=profile_workspaces.workspace_create,
```

Comment:
@JanVerderber tukaj imaš stvari čisto pomešane.

Imeti moraš dva URL-ja:

- `/workspaces` (GET): na ta url vežeš workspaces_list_handler. To si naredil, edina napaka, ki jo imaš pri tem URL-ju je endpoint (moral bi biti `profile.workspaces.workspaces_list_handler`)
- `/workspace/create` (POST): tukaj si dal napačen URL (dal si istega, kot za workspaces list). Tudi endpoint je napačen. `view_func` je pa okej.

Prosim popravi.